lib/rules/no-textarea-mustache.js

```diff
@@ -20,13 +20,21 @@ module.exports = {
   },
   /** @param {RuleContext} context */
   create(context) {
+    const sourceCode = context.getSourceCode()
+
     return utils.defineTemplateBodyVisitor(context, {
       /** @param {VExpressionContainer} node */
       "VElement[name='textarea'] VExpressionContainer"(node) {
         if (node.parent.type !== 'VElement') {
           return
         }
 
+        const parentStartTag = sourceCode.getText(node.parent.startTag)
```

Comment:
No need for the whole `sourceCode` here. You can use `node.parent.rawName` instead. See https://astexplorer.net/#/gist/d568be011d22bbc7c364e1eb4e2de668/b7f5155c779ee53528caa1373035b4d7436d4146

```suggestion
        const parentStartTag = node.parent.rawName
```